m_etcd/const.go

```diff
@@ -1,6 +1,9 @@
 package m_etcd
 
 const (
+	DefaultClaimTTL uint64 = 180 // 3 minutes in seconds
```

Comment:
I tried to explain it here: https://github.com/lytics/metafora/pull/145#issue-106874914

Basically raising it gives hearbeats longer to retry before they fail and shutdown the task (or entire node in the case of the node ttl). The higher the TTL the longer a work is running on a partitioned node before being gracefully shutdown.

So far the only time I've seen this hit is when etcd lost quorum and all metafora's gracefully shutdown. Kind of an ideal network partition, but in that case all a longer TTL would have done is potentially allowed nodes to survive the partition without restarting.